CMakeLists.txt

```diff
@@ -8,15 +8,14 @@ cmake_minimum_required(VERSION 3.15)
 cmake_policy(SET CMP0079 NEW)
 cmake_policy(SET CMP0091 NEW)
 
-set(version "1.4.0" CACHE STRING "Project version")
-
-string(REGEX MATCH "[0-9]+\\.[0-9]+\\.[0-9]" version_short ${version})
+set(BUILD_NUMBER "0" CACHE STRING "Build number")
+set(VERSION_STRING "" CACHE STRING "Project version (string)")
```

Comment:
This variable seems to be unused (other than printing it to the console)

Reply:
Sorry, I accidentally left it in.